src/app/src/main/kotlin/org/imperial/mrc/hint/models/CalibrateResultRow.kt

```diff
@@ -17,3 +17,28 @@ data class CalibrateResultRow(
     val upper: Number,
     @field:JsonProperty("area_level")
     val areaLevel: Int)
+
+data class FilterQuery(
+    val indicator: List<String>,
+    @field:JsonProperty("calendar_quarter")
+    val calendarQuarter: List<String>,
+    @field:JsonProperty("age_group")
+    val ageGroup: List<String>,
+    val sex: List<String>,
+    @field:JsonProperty("area_id")
+    val areaId: List<String>,
+    @field:JsonProperty("area_level")
+    val areaLevel: List<Int>)
+{
+    operator fun iterator(): Iterator<Pair<String, List<Any>>>
```

Comment:
This feels a bit like wanting to have your cake and eat it - defining what the fields are as a type, but then trying to treat the fields as if they're just any map to build the query. I think it would be better to have a slightly more verbose query that treats the fields separately. But that could be made neater by not building the SQL query string directly.